### PR TITLE
Feature/faq thermal

### PR DESCRIPTION
**Pull request by CEKrause**
Add FAQ following request from client services. Answer crowd sourced via Teams. 

* [x] I have spellchecked my content using Microsoft Word, Grammarly, or otherwise.
* [ ] If required, update the [DEA Tech Alerts and Changelog][TechAlertsChangelog] and the [DEA Sandbox Updates banner][SandboxUpdatesBanner].

[TechAlertsChangelog]: https://github.com/GeoscienceAustralia/dea-knowledge-hub/blob/main/docs/tech-alerts-changelog/_tech_alerts_changelog.md
[SandboxUpdatesBanner]: https://bitbucket.org/geoscienceaustralia/datakube-apps/src/64c28bbf3d0e019d8940547a22f78b9bfd58d739/clusters/dea-sandbox/sandbox.yaml

### CODE REVIEW

File: docs/guides/about/faq.md [hunk 1]
```diff
@@ -61,3 +61,28 @@ Table 1: Availability of the Landsat satellites for Earth observation and relate
 
 :::
 
+:::{dropdown} Why doesn't DEA make Landsat thermal bands available to users?
+:name: landsat-no-thermal
+
+Landsat satellites not only collect data in the short-wave spectrum, but also collect data through to the thermal 
+infrared bands. The USGS makes this data available in the form of a 
+[surface temperature and emissivity product](https://www.usgs.gov/landsat-missions/landsat-collection-2-surface-temperature), 
+which it provides separately to the surface reflectance products that DEA ingest to produce our [Analysis Ready Data (ARD) products](/data/category/dea-surface-reflectance/).
```
Comment (robbibt) [inline docs/guides/about/faq.md:70]: Missed this before - I think we use Level 1 Top of Atmosphere Radiance (TOA) as the input for our ARD, not surface reflectance. So maybe we could just remove "that DEA ingest to produce our [Analysis Ready Data (ARD) products](/data/category/dea-surface-reflectance/)."

File: docs/guides/about/faq.md [hunk 2]
```diff
@@ -61,3 +61,28 @@ Table 1: Availability of the Landsat satellites for Earth observation and relate
 
 :::
 
+:::{dropdown} Why doesn't DEA make Landsat thermal bands available to users?
+:name: landsat-no-thermal
+
+Landsat satellites not only collect data in the short-wave spectrum, but also collect data through to the thermal
```
Comment (robbibt) [inline docs/guides/about/faq.md:67]:
```suggestion
Landsat satellite sensors not only collect data in the short-wave spectrum, but also collect data through to the thermal 
```

File: docs/guides/about/faq.md [hunk 3]
```diff
@@ -61,3 +61,28 @@ Table 1: Availability of the Landsat satellites for Earth observation and relate
...
+
+The land surface temperature product that USGS produce is a global product, which uses global datasets to perform 
+corrections on the data that are collected by the satellite sensors. The land surface temperature outputs are very 
+sensitive to the atmospheric profile data that is used to perform the correction. For the global analysis, the 
+[NASA Modern Era Retrospective-Analysis for Research and Applications (MERRA)](https://gmao.gsfc.nasa.gov/reanalysis/MERRA-2/) 
+atmospheric data is used to provide values for height, air temperature and humidity. While this dataset works 
+well for a global correction, over Australia, studies over Australia have shown that the correction can be improved 
+when higher resolution datasets are considered ([Li et al, 2015](https://mssanz.org.au/modsim2015/L11/li.pdf)).  
+
+DEA’s ARD product provides high-quality data corrections for Australian conditions. At present, we are not producing a
```
Comment (robbibt) [inline docs/guides/about/faq.md:80]:
```suggestion
DEA’s ARD product provides high-quality data corrections for Australian conditions. At present, we do not produce a 
```

File: docs/guides/about/faq.md (hunk 1 above)
Comment (CEKrause) [reply docs/guides/about/faq.md]: Done. Thanks.

File: docs/guides/about/faq.md [hunk 4]
```diff
@@ -61,3 +61,28 @@ Table 1: Availability of the Landsat satellites for Earth observation and relate
...
+sensitive to the atmospheric profile data that is used to perform the correction. For the global analysis, the 
+[NASA Modern Era Retrospective-Analysis for Research and Applications (MERRA)](https://gmao.gsfc.nasa.gov/reanalysis/MERRA-2/) 
+atmospheric data is used to provide values for height, air temperature and humidity. While this dataset works 
+well for a global correction, over Australia, studies over Australia have shown that the correction can be improved 
+when higher resolution datasets are considered ([Li et al, 2015](https://mssanz.org.au/modsim2015/L11/li.pdf)).  
+
+DEA’s ARD product provides high-quality data corrections for Australian conditions. At present, we are not producing a 
+custom land surface temperature dataset for Australia, and so have not included the thermal bands in our ARD package. 
+
+If you do decide you’d like to make use of Landsat’s thermal data from the USGS directly, we have a
```
Comment (robbibt) [inline docs/guides/about/faq.md:83]:
```suggestion
If you would like to use Landsat thermal data provided by the USGS directly, we provide a
```

**Review by robbibt — APPROVED** [review]
Looks fantastic! The next one we should try and get in here is the "how is Australian and USGS ARD different" - would be a great pairing with this one!

**Comment by benji-glitsos-ga** [comment]
Wow, nice one Claire. I'll just quickly check this before we merge.

**Comment by CEKrause** [comment]
> Wow, nice one Claire. I'll just quickly check this before we merge.

I wanted to get Norman to check over this one before we merge, so hold off just yet.

**Comment by benji-glitsos-ga** [comment]
@CEKrause Here are the minor edits I made: https://github.com/GeoscienceAustralia/dea-knowledge-hub/compare/0581671e...017df616

**Comment by CEKrause** [comment]
Approval to publish given by Norman.

**Pull request merged**